app/build.gradle.kts

```diff
@@ -77,6 +77,9 @@ dependencies {
     implementation(libs.androidx.material3)
     implementation(libs.androidx.material.icons.extended)
     implementation(libs.androidx.lifecycle.viewmodel.compose)
+    //implementation(libs.androidx.appcompat)
+    //implementation(libs.material)
+    //implementation(libs.skiko.android)
```

Comment:
Perhaps we can remove these dependencies if they are not used.

Reply:
@yeroldsan Done